src/app/rundown-view/components/offsetable-piece/offsetable-piece.component.ts

```diff
@@ -57,6 +58,12 @@ export class OffsetablePieceComponent implements OnChanges, OnDestroy {
     }
   }
 
+  public get getTv2PieceMetadata(): Tv2PieceMetadata {
```

Comment:
Done

Reply:
This comment thread refers to two different options. It would be nice that you referred to which of the options you implemented (or even if it was a third option you found) instead of just "done".

Reply:
@LindvedKrvang this method is deleted from there and add it in piece-tooltip-component as:

  ```
public get tv2PieceType(): Tv2PieceType {
    const piece: Tv2Piece = this.piece as Tv2Piece
    return piece.metadata.type
  }
```